userProfile/src/components/custom-modal/custom-modal.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Button, Modal } from '@skbkontur/react-ui';
+
+type CustomModalProps = {
+    message: string[];
+    onClose: () => void;
+};
+
+const CustomModal = ({ message, onClose }: CustomModalProps) => {
+    return (
+        <Modal onClose={onClose}>
+            <Modal.Header>Пользователь сохранен</Modal.Header>
+            {message.length !== 0 && (
+                <Modal.Body>
+                    <p>Измененные данные:</p>
+                    {message.map(msg => (
+                        <p key={msg.length}>{msg}</p>
+                    ))}
+                </Modal.Body>
+            )}

```

Comment:
💥  и теперь нам не нужны эти проверки и нет цикла в верстке, красота)
```suggestion
    <Modal.Body>
                <p>Измененные данные:</p>
                {getModalMessages(formData.state, formData.prevState)}
            </Modal.Body>
```